packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html

```diff
@@ -1,59 +1,54 @@
-<div class="cdn-dedicated-domain">
-    <!-- Header -->
-    <div class="page-header">
-        <h1 class="oui-header_2">
+<header class="oui-page-header">
+    <div class="oui-page-header__container">
+        <h1 class="oui-page-header__title">
             <span data-ng-bind="$ctrl.domain.domain"></span>
             <span
                 class="label d-inline-block align-middle ml-3 oui-status"
                 data-ng-bind="$ctrl.domain.mode"
                 data-ng-class="{
-                      'oui-status_success': $ctrl.domain.mode === 'ON',
-                      'oui-status_error': $ctrl.domain.mode ==='OFF'
-                  }"
+                    'oui-status_success': $ctrl.domain.mode === 'ON',
+                    'oui-status_error': $ctrl.domain.mode ==='OFF'
+                }"
             >
             </span>
         </h1>
     </div>
-    <!-- End of Header -->
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            text="{{:: 'cdn_domain_tab_statistics' | translate }}"
+            state="app.networks.cdn.dedicated.domain.statistics"
```

Comment:
Could have been nice to extract those through the router's resolve if possible

Reply:
Yes, could have 🌞